packages/wallet/backend/src/cardUsers.ts

```diff
@@ -0,0 +1,110 @@
+import { createContainer } from '@/createContainer'
+import { env } from '@/config/env'
+import { ICreateCustomerRequest } from '@/card/types'
+
+interface UserData {
+  email: string
+  firstName: string
+  lastName: string
+  ppNumber: string
+}
+
+const entries = `John;Doe;john@doe.com:8888
+Alice;Smith;alice@smith.com;9999`
+
+function processEntries() {
+  const users: Array<UserData> = []
+  const lines = entries.split('\n')
+  for (const line of lines) {
+    const [firstName, lastName, email, ppNumber] = line.split(';')
+    users.push({
+      email,
+      firstName,
+      lastName,
+      ppNumber
+    })
+  }
+
+  return users
+}
+
+async function cardManagement() {
+  const container = await createContainer(env)
+  const logger = container.resolve('logger')
+  const knex = container.resolve('knex')
+  const gateHubClient = container.resolve('gateHubClient')
+  const accountProductCode = env.GATEHUB_ACCOUNT_PRODUCT_CODE
+  const cardProductCode = env.GATEHUB_CARD_PRODUCT_CODE
+  const nameOnCardPrefix = env.GATEHUB_NAME_ON_CARD_PREFIX
+  const GATEWAY_UUID = env.GATEHUB_GATEWAY_UUID
+
+  try {
+    const usersData = processEntries()
+
+    for (const userData of usersData) {
+      const { email, firstName, lastName, ppNumber } = userData
+
+      const managedUser = await gateHubClient.createManagedUser(email)
+
+      logger.info(`Created managed user for ${email}: ${managedUser.id}`)
+
+      await gateHubClient.connectUserToGateway(managedUser.id, GATEWAY_UUID)
+
+      logger.info(
+        `Connected user ${managedUser.id} - ${managedUser.email} to gateway`
+      )
+
+      const user = await gateHubClient.getWalletForUser(managedUser.id)
+      const walletAddress = user.wallets[0].address
```

Comment:
should we use the default wallet created instead of creating on for each account?

Reply:
`getWalletForUser` retrieves the already created account - when the managed user is created. We are not creating another wallet here.